docs/configuration/config-file-reference.md

```diff
@@ -2217,17 +2217,18 @@ sharding_ring:
   [wait_active_instance_timeout: <duration> | default = 10m]
 
 # The compaction strategy to use. Supported values are: default, partitioning.
-# CLI flag: -compactor.compaction-mode
-[compaction_mode: <string> | default = "default"]
+# CLI flag: -compactor.compaction-strategy
+[compaction_strategy: <string> | default = "default"]
 
-# How long block visit marker file should be considered as expired and able to
-# be picked up by compactor again.
-# CLI flag: -compactor.block-visit-marker-timeout
-[block_visit_marker_timeout: <duration> | default = 5m]
+# How long compaction visit marker file should be considered as expired and able
+# to be picked up by compactor again.
+# CLI flag: -compactor.compaction-visit-marker-timeout
+[compaction_visit_marker_timeout: <duration> | default = 1m30s]
```

Comment:
Isn't it safer to leave it higher? Do you you what happens if the `compaction-visit-marker-file-update-interval` fails? Will it retry in the next 1min or will it fail the compaction. I am worry that it can fail once and compaction is already marked as expired and two compactor can run the same compaction job.

Reply:
Good point. How about set default value to 10 minutes?